go.work

```diff
@@ -1,3 +1,6 @@
 go 1.20
 
-use ./relayer
+use (
+	./go-ethereum
```

Comment:
Just curious why this is change necessary? I am not too familiar with `go.work` files.

Reply:
Not necessary just removed.